blivet/__init__.py

```diff
@@ -127,8 +127,6 @@ def enable_installer_mode():
     from pyanaconda.anaconda_log import program_log_lock
     util.program_log_lock = program_log_lock
 
-    flags.installer_mode = True
-
```

Comment:
It would be nice if this whole function could be replaced by some ``Config`` object the attributes of which would be the things imported above set from the Anaconda and passed to ``Blivet()`` (or ``InstallerStorage()`` of course).